src/toast/ops/mapmaker_utils/mapmaker_utils.py

```diff
@@ -689,12 +689,51 @@ def _exec(self, data, detectors=None, use_accel=None, **kwargs):
 
         if use_accel:
             if not zmap.accel_exists():
+                # Does not yet exist, create it
                 log.verbose_rank(
-                    f"Operator {self.name} zmap not yet on device, copying in",
+                    f"Operator {self.name} zmap not yet on device, creating",
+                    comm=data.comm.comm_group,
+                )
+                zmap.accel_create(f"{self.name}")
+                zmap.accel_used(True)
```

Comment:
this could use a create with `zero_out=True`

Reply:
Fixed, and I added the needed `zero_out` argument to `PixelData._accel_create()`.

Reply:
Will `accel_create` forward the argument to `_accel_create`?